log/term/term.go

```diff
@@ -7,11 +7,23 @@ import (
 	"github.com/go-kit/kit/log"
 )
 
+type parentWriter interface {
+	ChildWriter() io.Writer
+}
+
+func resolveWriter(w io.Writer) io.Writer {
```

Comment:
Note: This method is a pass-through for all  `io.Writer` instances that do not implement the `parentWriter` interfacem, thus **not** breaking the existing behavior.